subworkflows/local/gene_annotation.nf

```diff
@@ -0,0 +1,75 @@
+// INCLUDING MODULES
+// Importing the necessary modules for the workflow
+include { BAKTA_BAKTA } from '../../modules/nf-core/bakta/bakta/main'
+include { BAKTA_BAKTADBDOWNLOAD } from '../../modules/nf-core/bakta/baktadbdownload/main'
+include { PROKKA } from '../../modules/nf-core/prokka/main'
+
+// Defining the workflow
+workflow GENE_ANNOTATION{
+    // Defining the input channel
+    take:
+        ch_genome
+
+    // Defining the main process
+    main:
+        // Initializing empty channels for the output files
```

Comment:
need to initialize channels for `prokka_fna`, `prokka_faa`, `bakta_fna`, `bakta_faa`

Reply:
resolved with [39ac2ca](https://github.com/cidgoh/nf-seqqc/pull/44/commits/39ac2ca1315a9cf9005206e59ddc84265cc88b21)